### PR TITLE
Add built xgboost-operator image to README

### PR DESCRIPTION
**Pull request by XinkaiWang515**
Add built xgboost-operator image to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,3 +193,16 @@ Events:
...
+# Docker Images
+
+You can use the following Dockerfiles to build the images yourself:
+
+* [xgboost-operator](https://github.com/kubeflow/xgboost-operator/blob/master/Dockerfile)
+
+For your convenience, you can download built xgboost-operator image by:
+
+```
+docker pull stevendoit/xgboost-operator:1.0
```
Comment (merlintang) [inline README.md:207]: @terrytangyuan we would recommend to push image to gcr.io/kubeflow-images-public/xgboost-operator:v1.0. but Xinkai does not own the push right. how can we handle it ?

**Comment by merlintang** [comment]
fix this https://github.com/kubeflow/xgboost-operator/issues/58

File: README.md (hunk 1 above)
Comment (terrytangyuan) [reply README.md:207]: Could you use this link? See my comment in https://github.com/kubeflow/xgboost-operator/issues/58#issuecomment-626382739

**Comment by merlintang** [comment]
does this link publics.t eveythone can download it? I can not access this
link. :)

**Comment by terrytangyuan** [comment]
Yes it’s public. Everyone should be able to access it.

**Pull request closed without merging**